applications/derive_mirror_rnda.py

```diff
@@ -142,15 +152,25 @@ def plotMeasuredDistribution(file, **kwargs):
     parser.add_argument(
         "-m",
         "--model_version",
-        help="Model version (default=prod4)",
+        help="Model version (default=Current)",
         type=str,
-        default="prod4",
+        default="Current",
+    )
+    parser.add_argument(
+        "--containment_mean",
+        help="Mean of measured containment diameter [cm]",
+        type=float, required=True
     )
     parser.add_argument(
-        "--mean_d80", help="Mean of measured D80 [cm]", type=float, required=True
+        "--containment_sigma",
+        help="Std dev of measured containment diameter [cm]",
+        type=float, required=False
     )
     parser.add_argument(
-        "--sig_d80", help="Std dev of measured D80 [cm]", type=float, required=False
+        "--containment_fraction",
+        help="Containment fraction for diameter calculation (in interval 0,1)",
```

Comment:
Suggest to add [], i.e., "in interval [0, 1]".
See comment below about another option instead.